src/Cassandra.IntegrationTests/Ado/AdoBasicTests.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace Cassandra.IntegrationTests.Data
 {
-    [TestFixture, Category("short")]
+    [TestFixture, Category("short"), Category("realcluster")]
```

Comment:
I really like the idea to have a category where tests can share a cluster.

We could use a category name that carries more meaning than "realcluster"? Some suggestions:
- minimal
- basic
- singlecluster

Reply:
Its hard to name categories that serve every purpose and `realcluster` allows us to mark those that require C* clusters.
I was thinking about `minimal`, as a subgroup of `short` that would be for the 5 min runs in appveyor and also allows us to smoke test new versions of the server, but there isn't much overlap (simulacron doesn't test the server 😃 )

Nevermind, sorry for the misunderstanding.

Reply:
Sounds good!